packages/tree-select/package.json

```diff
@@ -29,6 +29,6 @@
 	"scripts": {
 		"clean": "npx rimraf dist",
 		"prepublish": "npm run clean",
-		"prepare": "npx @automattic/calypso-build"
+		"prepare": "../calypso-build/bin/transpile.js"
```

Comment:
Ugly but effective 😅

Reply:
How was that broken? :thinking:

Reply:
(Fine to keep the ugly-but-effective version to be able to ship this soon.)

Reply:
I may have been wrong and "fixed" this before doing the `distclean` dance to get the monorepo package bins properly installed.

Please double check me, we may be able to use `npx --package @automattic/calypso-build transpile`, although I suspect the change will be painful for some folks as they'll likely experience the same problem.

Reply:
Wondering if #32383 might've been the actual reason for this. Seems like a stretch, but maybe some silent error when trying to build a local package along the dependency chain?

Reply:
I reverted the commit since it seems to work in my testing :thinking: I'll leave it to reviewers/testers to correct me there...